lib/abacus.js

```diff
@@ -108,3 +108,8 @@ Abacus.prototype.debug = function() {
   var log = this.opts.debug;
   log.apply(log, arguments);
 };
+
+Abacus.prototype.stop = function() {
```

Comment:
cool, maybe should `this.stopFlushing()` as well to clear the interval? Or, we should dereference it on creation.